storage/innobase/xtrabackup/doc/source/advanced/encrypted_innodb_tablespace_backups.rst

```diff
@@ -182,3 +185,146 @@ Preparing the second incremental backup is a similar process: apply the deltas t
 
 Backup is now prepared and can be restored with :option:`xtrabackup --copy-back` option. In case the keyring has been rotated you'll need to restore the keyring which was used to take and prepare the backup.
 
+Backups with ``keyring_vault`` plugin
```

Comment:
This entire section really work for server with both plugins, i.e. it is perfectly fine to create a backup with command
`xtrabackup --backup --user=root -p --target-dir=/data/backup --transition-key=MySecetKey` when server is using `keyring_file` plugin. What really has changed is
1. Now there is more than one plugin, this part is clear from the "Two keyring plugins supported by |xtrabackup| are ``keyring_file`` and ``keyring_vault``" and is fine by me
2. Previously, the only supported case was the one described below in "Creating Backup", i.e. xtrabackup required access to the keyring during the backup and prepare, and it was on the user to make sure that keyring file is accessible (now that we added keyring_vault, it is on user to make sure vault server is accessible). The cases described in "backups with passphrase" and "with stored passphrase" also apply to keyring file.

Instead of sections
1. keyring file
2. keyrig vault
  - Creating Backup
  - backups with passphrase
  - backups with stored passphrase

There should be sections:
1. Creating Backup
2. backups with passphrase
3. backups with stored passphrase

With some specifics explained in each regarding how to deal with different plugins. Like when do we need to copy keyring file and when do we need to make sure that xtrabackup is able to connect to keyring vault server.